rotkehlchen/assets/spam_assets.py

```diff
@@ -94,10 +103,15 @@ def query_token_spam_list() -> Set[EthereumToken]:
             tokens_to_ignore.add(token)
 
     # Try to add custom list
-    for token_identifier in KNOWN_ETH_SPAM_TOKENS:
+    for token_address in KNOWN_ETH_SPAM_TOKENS:
         try:
-            own_token = EthereumToken.from_identifier(token_identifier)
-        except UnknownAsset:
+            own_token = get_or_create_ethereum_token(
+                userdb=db,
+                ethereum_address=token_address,
+                protocol=SPAM_PROTOCOL,
```

Comment:
kek. This may actually come to bite us in the future if there is a protocol called `spam`

Reply:
As discussed we will go with this for now but may need to change it in the future.